### ISSUE TITLE
socket.io is not controlled by HOST_URL

### ISSUE THREAD

**Issue by fufenghua** [zilliztech/attu#525]
`
version: '3.8'        
                      
services:             
  attu:               
    image: zilliz/attu:v2.4.0
    container_name: attu
    restart: always   
    hostname: attu       
    environment:         
      - MILVUS_URL=10.15.85.78:19530
      - HOST_URL=/attu                                                                                                                        
    ports:            
      - 5949:3000
 `

location /attu/ {
            proxy_set_header Host $http_host;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header X-Forwarded-Proto $scheme;
            proxy_pass http://10.15.86.23:5949/;
            proxy_set_header Upgrade $http_upgrade;
            proxy_set_header Connection "Upgrade";
}

With this configuration, the API interface is normal, but socket.io is abnormal

![29cdecd95d7a97ce311d3399aeea347](https://github.com/zilliztech/attu/assets/18100287/224756de-17a6-43ac-adec-c425577290d8)

**Comment by shanghaikid** [comment]
```nginx
server {
  listen 8080;
  server_name localhost;

  
  location /attu/ {
    proxy_pass http://localhost:3000/;
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "upgrade";
    proxy_set_header Host $host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;
    rewrite ^/attu/(.*)$ /$1 break;
  }
}
```

This is a working configuration.  I don't think you need to define the `HOST_URL` as `/attu`, just define it as the ip or the server name.

**Comment by shanghaikid** [comment]
Yes, it should be fixed in the v2.4.4, please upgrade then follow this doc. 

https://github.com/zilliztech/attu/blob/main/doc/use-attu-behind-proxy.md
